datumaro/plugins/yolov8_format/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 vincent
```

Comment:
vincent? Apache License?

Reply:
Unintended. This was a VS Code extension glitch. Removed